tests/unit/test_recipes.py

```diff
@@ -0,0 +1,22 @@
+"""Recipe tests."""
+from pathlib import Path
+import pytest
+import yaml
+
+from tests.integration.test_recipes_loading import RECIPES, IDS
+import esmvaltool
+
+REFERENCES_PATH = Path(esmvaltool.__file__).absolute().parent / 'references'
+
+
+@pytest.mark.parametrize('recipe_file', RECIPES, ids=IDS)
+def _test_reference_tags(recipe_file):
+    recipe = yaml.safe_load(recipe_file.read_text())
+    value = recipe['documentation']['references']
+    _check_bibtex_exist(value)
+
+
+def _check_bibtex_exist(tag):
+    """Check bibtex file is added to REFERENCES_PATH."""
+    bibtex_file = REFERENCES_PATH / f'{tag}.bibtex'
+    assert not bibtex_file.is_file()
```

Comment:
```suggestion
def test_reference_tags(recipe_file):
    recipe = yaml.safe_load(recipe_file.read_text())
    tags = recipe.get('documentation', {}).get('references', [])
    for tag in tags:
        bibtex_file = REFERENCES_PATH / f'{tag}.bibtex'
        assert not bibtex_file.is_file()
```
- The name of the function has to start with `test_` so [`pytest` knows it is a test](https://docs.pytest.org/en/latest/goodpractices.html#conventions-for-python-test-discovery).
- It would be good to write the test in such a way that recipes without a documentation/references list do not cause the test to fail, because that is not what we are checking here
- Note that the content of `references` in the recipe is a list of strings, not a single string
- It would be great if you could make a nice error message with a link to the documentation on adding a bibtex file to esmvaltool/references

Reply:
@bouweandela Thanks a lot. the error message is added. Also, running this test shows that some bibtex files are missing in the references folder. I added those. 

@mattiarighi there are two tags in the current [config-references.yml](https://github.com/ESMValGroup/ESMValTool/blob/master/esmvaltool/config-references.yml):  `cds-satellite-albedo`, and `mls-aura`. I couldn't find any citation information about the first tag. It seems that the provided link is for data in zip format rather than a reference entry.  For the second one, I made its [bibtex file](https://github.com/ESMValGroup/ESMValTool/pull/1545/files#diff-0412b64adf74ea02a1ccdde0a5d7401c). Could you please check these two tags? Thanks.